kalite/topic_tools/__init__.py

```diff
@@ -460,8 +460,36 @@ def get_assessment_item_data(request, assessment_item_id=None):
 
     # TODO (rtibbles): Enable internationalization for the assessment_items.
 
+    try:
+        logging.info('Wrapping the Assessment Item with _()')
+        item_data = json.loads(assessment_item['item_data'])
+        question_content = _(item_data['question']['content'])
+        answerarea_content = item_data['answerArea']['options']['content']
+        if answerarea_content != "":
+             answerarea_content = _(answerarea_content)
+
+        # Loop over multiple hints contents.
+        # Pickup the wrapped strings and append to hints dict
+        new_hints = []
+        for contents_dict in item_data['hints']:
+            content_value = contents_dict.get('content')
+            converted_content = _(content_value)
+            contents_dict['content'] = converted_content
+            new_hints.append(contents_dict)
+
+        item_data['hints'] = new_hints
+        item_data['question']['content'] = question_content
+        item_data['answerArea']['options']['content'] = answerarea_content
+        # dump data to make it to a proper json format.
+        assessment_item['item_data'] = json.dumps(item_data)
+        logging.info('Successfully wrapped New Assessment Item with _()')
+
+    except KeyError:
+        logging.log("Something wrong with the format on assessment items json.")
```

Comment:
1. This needs to be more informative: include the error message from the exception itself.
2. Fix the string: `"There is something wrong with the format of the assessment items: %s"`
3. Use `logging.error`. Or `logging.warning` if this is not a catastrophic error.